hugo_stats.json

```diff
@@ -105,6 +105,9 @@
       "content-dropdown",
       "cta",
       "cursor-pointer",
+      "dar",
```

Comment:
I guess these 3 are not necessary.
And as I understand this file is generated each time with **npm run start**. So, maybe we can put it to .gitignore.